community/concepts/grain.md

```diff
@@ -0,0 +1,73 @@
+### Grain
+
+SourceCred is both a social algorithm for assigning [cred](concepts/cred.md)
```

Comment:
This is within the concepts folder already, so the path should be relative to that.

Reply:
Fixed